homeassistant/components/switchbot/sensor.py

```diff
@@ -72,6 +73,13 @@
         state_class=SensorStateClass.MEASUREMENT,
         device_class=SensorDeviceClass.TEMPERATURE,
     ),
+    "power": SensorEntityDescription(
+        key="power",
+        name="Power",
+        native_unit_of_measurement=POWER_WATT,
```

Comment:
`POWER_WATT` is deprecated. Use `UnitOfPower.WATT`.

Reply:
Thank you.

I have fixed it.